README.md

```diff
@@ -73,6 +73,10 @@ jobs:
         auto-merge: true # If `true`, will try to automatically merge the pull requests.
 ```
 
+### Output
+When triggered by a push event, the pull request number which is created or found to
+already exist is stored in the `pull-request` output variable.
```

Comment:
Please don't break markdown lines like that. If you're bothered by the horizontal scrollbar, consider activating the word wrap in your text editor (`ALT+Z`, if you're using VSCode 🙂).

Reply:
My editor did this without me noticing. Will correct.